### ISSUE TITLE
Medusa has started using alot of API hits

### ISSUE THREAD

**Issue by jbaun** [pymedusa/Medusa#7116]
**Describe the bug**

For some reason Medusa has started using alot of API hits, and my provider has a limit of 250 hits pr day. I have nothing in the backlog and no new episodes in "wanted", but for some resason it is using over 100 api hits doing the time of the attached log.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Medusa (please complete the following information):**
Medusa Info:	Branch: master
Commit: 74c3f124fb599ad923e23f46fc320778b8b4217c
Version: 0.3.4
Database: 44.14
 Python Version:	2.7.14 (default, May 28 2018, 14:47:12) [GCC 4.6.4]
 SSL Version:	OpenSSL 1.0.2n 7 Dec 2017
 OS:	Linux-2.6.32.12-armv5tel-with-glibc2.4
 Locale:	Unknown.Unknown

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
2019-09-06 09:45:09 INFO     POSTPROCESSOR :: [74c3f12] /volume1/downloads/complete/tv/.stfolder: Hidden or ignored folder
2019-09-06 09:45:09 INFO     POSTPROCESSOR :: [74c3f12] I did encounter some unprocessable items: 
2019-09-06 09:45:09 INFO     POSTPROCESSOR :: [74c3f12] Successfully processed.
2019-09-06 09:44:03 INFO     FINDPROPERS :: [74c3f12] Completed the search for new propers, next check in approx. 3h, 58m
2019-09-06 09:43:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: [74c3f12] No needed episodes found
2019-09-06 09:43:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: [74c3f12] Beginning daily search for new episodes
2019-09-06 09:42:57 INFO     FINDPROPERS :: [Danishbits] :: [74c3f12] Searching for any new PROPER releases from Danishbits
2019-09-06 09:42:57 INFO     FINDPROPERS :: [74c3f12] Using proper search days: 2
2019-09-06 09:42:57 INFO     FINDPROPERS :: [74c3f12] Beginning the search for new propers
2019-09-06 09:42:55 INFO     BACKLOG :: [74c3f12] Running full backlog search on missed episodes for selected shows
```
</details>

**Additional context**

**Comment by medariox** [comment]
The search for PROPER releases happens also if you don't have any needed episodes. Please post a **complete debug log**.

**Comment by jbaun** [comment]
Ahh yes sorry. I forgot to look at the dubug log. Now i understand what the problem is.
I have some shows where there are some preair episodes, that Medusa has already processe, but because the airdate is in the future Medusa is still using 4 api hits for every of these episodes, even though they are not wanted:

2019-09-06 09:44:03 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] GET URL: https://danishbits.org/couchpotato.php?search=Undskyld+Norge+S01E08+RERIP&latest=false&user=**********&passkey=********** [Status: 200]
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search string: Undskyld Norge S01E08 RERIP
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search mode: Episode
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] User-Agent: Medusa/0.3.4 (Linux; 2.6.32.12; 9ff084f3-cfb0-11e9-998d-0011320f9e92)
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] GET URL: https://danishbits.org/couchpotato.php?search=Undskyld+Norge+S01E08+REAL&latest=false&user=**********&passkey=********** [Status: 200]
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search string: Undskyld Norge S01E08 REAL
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search mode: Episode
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] User-Agent: Medusa/0.3.4 (Linux; 2.6.32.12; 9ff084f3-cfb0-11e9-998d-0011320f9e92)
2019-09-06 09:44:02 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] GET URL: https://danishbits.org/couchpotato.php?search=Undskyld+Norge+S01E08+REPACK&latest=false&user=**********&passkey=********** [Status: 200]
2019-09-06 09:44:01 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search string: Undskyld Norge S01E08 REPACK
2019-09-06 09:44:01 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search mode: Episode
2019-09-06 09:44:01 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] User-Agent: Medusa/0.3.4 (Linux; 2.6.32.12; 9ff084f3-cfb0-11e9-998d-0011320f9e92)
2019-09-06 09:44:01 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] GET URL: https://danishbits.org/couchpotato.php?search=Undskyld+Norge+S01E08+PROPER&latest=false&user=**********&passkey=********** [Status: 200]
2019-09-06 09:44:01 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search string: Undskyld Norge S01E08 PROPER
2019-09-06 09:44:01 DEBUG    FINDPROPERS :: [Danishbits] :: [74c3f12] Search mode: Episode

**Comment by jbaun** [comment]
So maybe i just need to disabled "Download propers"

**Comment by medariox** [comment]
Yeah, we need to improve the Proper search. It currently requires a lot of hits. E.g. if you have 4 epsiodes that need proper, with the default settings: 4 * 4 * 6 = 96 hits.

**Comment by jbaun** [comment]
ok. good. thank you for quick reply :)
